policies/constraints/README.md

```diff
@@ -1 +0,0 @@
-This directory should contain the constraint .yaml files used in your environment. You can find sample constraints under https://github.com/GoogleCloudPlatform/policy-library/tree/master/samples.
```

Comment:
Why is this being removed? It keeps the directory and is a helpful pointer IMO.

Reply:
If you run the bundler and sink it to the `policies\constraints` folder as mentioned in the README.md, it fails since the directory is already there. We could keep this and add a subfolder to `policies\constraints` as `policies\constraints\$BUNDLE`. That should work.

Reply:
That would probably be better.

Reply:
Acked.